### PR TITLE
chore: version bump Taskgraph 4.1.2

### PR DESCRIPTION
**Pull request by hneiva**

### CODE REVIEW

File: version.txt [hunk 1]
```diff
@@ -1 +1 @@
-4.1.0
+4.2.0
```
Comment (ahal) [inline version.txt:1]: I think we should bump to `4.1.1` as a minor upgrade would imply there's some sort of new feature.

**Review by ahal — CHANGES_REQUESTED** [review]

File: version.txt (hunk 1 above)
Comment (hneiva) [reply version.txt:1]: Works for me

**Review by ahal — APPROVED** [review]
Thanks! Note the commit message still says 4.2.0

**Comment by ahal** [comment]
Superceded by https://github.com/taskcluster/taskgraph/pull/200

**Pull request closed without merging**